x/slashing/keeper/keeper.go

```diff
@@ -29,8 +29,8 @@ type Keeper struct {
 	Schema                     collections.Schema
 	Params                     collections.Item[types.Params]
 	ValidatorSigningInfo       collections.Map[sdk.ConsAddress, types.ValidatorSigningInfo]
-	AddrPubkeyRelation         collections.Map[[]byte, cryptotypes.PubKey]
-	ValidatorMissedBlockBitmap collections.Map[collections.Pair[[]byte, uint64], []byte]
+	AddrPubkeyRelation         collections.Map[[]byte, cryptotypes.PubKey]               // key: PubKey address | value: PubKey
```

Comment:
I haven't verified, but should the comments be in the form:
`// AddrPubkeyRelation key: PubKey address | value: PubKey` above the field, to show properly and to look better?

Reply:
I could do that, but I elaborated some of them for better understanding

Reply:
I actually meant above the field, like

```go
// AddrPubkeyRelation lorem ipseum
AddrPubkeyRelation         collections.Map[[]byte, cryptotypes.PubKey]
```